.github/workflows/stale.yaml

```
@@ -0,0 +1,32 @@
+name: 'Label and close stale PRs'
+on:
+  schedule:
+    # Run at 1:30 AM UTC every day
+    - cron: '30 1 * * *'
+  workflow_dispatch: # Allows manual triggering
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v9
+        with:
+          # Pull Request Settings
+          days-before-pr-stale: 150
+          days-before-pr-close: 30
+          stale-pr-message: 'Thank you for your contribution. This pull request has been marked as stale because it has had no activity in the last 150 days. It will be closed in 30 days if there is no further activity. If you need more time, you can remove the "stale" label.'
```

> If you need more time, you can remove the "stale" label.

Can a non-maintainer even remove the label? At least I don't seem to have permissions to change labels for my own PRs in repositories, where I'm not an org member (i.e. outside `grafana` org)?

---

good call. They can't

```suggestion
          stale-pr-message: 'Thank you for your contribution. This pull request has been marked as stale because it has had no activity in the last 150 days. It will be closed in 30 days if there is no further activity. If you need more time, you can add a comment to the PR.'
```